### PR TITLE
[16.0] [MIG] sale_payment_sheet

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

File: sale_payment_sheet/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "Sale payment sheet",
     "summary": "Allow to create invoice payments to commercial users without "
     "accounting permissions",
-    "version": "15.0.1.1.2",
+    "version": "16.0.1.1.2",
```
Comment (FrancoMaxime) [inline sale_payment_sheet/__manifest__.py]: Hi, you should restart the version from 16.0.1.0.0
```suggestion
    "version": "16.0.1.0.0",
```

File: sale_payment_sheet/tests/test_sale_payment_sheet.py [hunk 2]
```diff
@@ -76,7 +91,7 @@ def _create_payment_sheet(self):
                 with sheet_form.line_ids.new() as line_sheet:
                     line_sheet.partner_id = self.partner
                     line_sheet.invoice_id = invoice
-                    line_sheet.ref = "REF{}".format(line_sheet.id)
+                    # line_sheet.ref = "REF{}".format(line_sheet.id)
```
Comment (FrancoMaxime) [inline sale_payment_sheet/tests/test_sale_payment_sheet.py]: Why this line is commented?

**Review by FrancoMaxime — CHANGES_REQUESTED** [review]
Hi, code LGTM, minors comment

File: sale_payment_sheet/tests/test_sale_payment_sheet.py (hunk 2 above)
Comment (angelmoya) [reply sale_payment_sheet/tests/test_sale_payment_sheet.py]: Ref in view is not accesible, with group="base.group_no_one", in this version we can't get this field, for other fields I add full accounting features to the user, but I don't know why for this field and group thid doesn't work

File: sale_payment_sheet/__manifest__.py (hunk 1 above)
Comment (angelmoya) [reply sale_payment_sheet/__manifest__.py]: ok fixed

File: sale_payment_sheet/tests/test_sale_payment_sheet.py (hunk 2 above)
Comment (angelmoya) [reply sale_payment_sheet/tests/test_sale_payment_sheet.py]: I just delete instead of coment

**Review by aliciagaarzo — APPROVED** [review]

**Review by FrancoMaxime — APPROVED** [review]
LGTM, thanks

File: sale_payment_sheet/tests/test_sale_payment_sheet.py [hunk 3]
```diff
@@ -60,18 +61,30 @@ def _create_invoice(self):
                 line_form.price_unit = 100.00
         return invoice_form.save()
 
+    def _create_refund(self):
+        with Form(
```
Comment (pedrobaeza) [inline sale_payment_sheet/tests/test_sale_payment_sheet.py:65]: Don't do `with` here, as you are saving 2 times with it (one exiting the context and the other on the explicit call).

**Review by pedrobaeza — COMMENTED** [review]
Please squash your last commit with the migration one, indicating that it also improves tests, or put the version change in the migration commit if you want to preserve both commits.

File: sale_payment_sheet/tests/test_sale_payment_sheet.py (hunk 3 above)
Comment (rousseldenis) [reply sale_payment_sheet/tests/test_sale_payment_sheet.py:65]: @angelmoya

**Comment by pedrobaeza** [comment]
@carolinafernandez-tecnativa please supersede this PR fixing my last commit to proceed with the merge.

**Comment by pedrobaeza** [comment]
Superseded by #2699

**Pull request closed without merging**